integration/integration_test.py

```diff
@@ -1048,10 +1114,33 @@ def file_to_stream(self, file_path, stream_path):
             print(' '.join(cmd))
         run_cmd(cmd)
 
+    def flight_request(self, json_path, arrow_path):
+        cmd = ['java', '-cp', self.ARROW_FLIGHT_JAR,
+               'org.apache.arrow.flight.example.integration.IntegrationTestClient',
+               '-j', json_path, '-a', arrow_path]
+        if self.debug:
+            print(' '.join(cmd))
+        run_cmd(cmd)
+
+    @contextlib.contextmanager
+    def flight_server(self):
+        cmd = ['java', '-cp', self.ARROW_FLIGHT_JAR,
+               'org.apache.arrow.flight.example.integration.IntegrationTestServer']
+        if self.debug:
+            print(' '.join(cmd))
+        server = subprocess.Popen(cmd)
+        try:
+            time.sleep(1)
+            yield
+        finally:
+            server.terminate()
```

Comment:
Ideally you would add `server.wait()` after this. Unless it slows down the tests too much?

Reply:
I'll give it a try. The Java tests are already quite slow; I want to dig into why it seems to take so long to just start the server.

Reply:
Apparently I'm hitting https://github.com/netty/netty/issues/6454, so it may or may not be slow for others.